contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -147,33 +164,27 @@
         }
       },
       async handleSave() {
-        this.validate();
-        if (this.error) {
-          return;
-        }
-
         await Promise.all(
-          this.nodes.map(node => {
+          this.nodes.map(async node => {
             const fieldValue = {};
-            Object.entries(this.selectedValues).forEach(([key, value]) => {
-              if (value.includes(node.id)) {
+            const currentNode = this.getContentNode(node.id);
+            if (this.hasMixedCategories) {
+              Object.assign(fieldValue, currentNode[this.field] || {});
+            }
+
+            Object.entries(this.selectedValues)
+              .filter(([value]) => value.length === this.nodeIds.length)
+              .forEach(([key]) => {
                 fieldValue[key] = true;
-              }
-            });
-            if (this.updateDescendants && node.kind === ContentKindsNames.TOPIC) {
-              return this.updateContentNodeDescendants({
-                id: node.id,
-                [this.field]: fieldValue,
               });
-            }
             return this.updateContentNode({
               id: node.id,
               [this.field]: fieldValue,
             });
           })
         );
         this.$store.dispatch('showSnackbarSimple', this.confirmationMessage || '');
-        this.close(this.changed);
+        this.close();
```

Comment:
It seems we still have the removed change. We need to have this here
```suggestion
        this.close(this.changed);
```